containerized_analytics/smile/docker-compose-smile.yml

```diff
@@ -270,13 +259,12 @@ services:
     restart: unless-stopped
 
   collect-reddit-comment:
-    image: socialmediamacroscope/collect_reddit_comment:0.1.2
+    image: socialmediamacroscope/collect_reddit_comment:latest
```

Comment:
Let's thinks about using the latest tag one more time. Should we use the certain version that are stable, then whenever, the new stable version are ready, update these yml files and release the new version of this repo instead of just using the latest? Just suggesting

Reply:
Sounds good to me.

Reply:
On another thought, since we will try config.yaml at some point, might as well just leave them latest for now...